### DISCUSSION TITLE
[Feature Request]: Support for API V1. Guys! Let's support the API

### DISCUSSION THREAD

**Discussion by soltanoff** [acheong08/ChatGPT#1135] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Hi! 
Thank you very much for your work!
And I have a question: How can I buy you coffee? (for [example](https://www.buymeacoffee.com/))

I guess I'm not the only one who wants to support you. :) 

### Proposed workflow

Press F to pay respect.

**Comment by jianhai0527** [comment]
+1

**Comment by 7trail** [comment]
Agreed. You deserve a ton of support for the work you've done on a 20K+ starred repo.

**Comment by hrizal** [comment]
+1

**Comment by tommyxps** [comment]
+1

**Comment by acheong08** [comment]
Money isn't my main problem but whether I can spend it. I have around $9k stuck in USDT but no platforms are willing to let me withdraw it. I also earn a ton from security bounties but all that is held by my parents so I can't spend it.

**Reply by hrizal** [reply]
https://u.today/3-million-worth-tether-usd-usdt-account-has-just-been-frozen-whats-going-on ?
